pkg/license/reporter.go

```diff
@@ -0,0 +1,73 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package license
+
+import (
+	"time"
+
+	"github.com/elastic/cloud-on-k8s/pkg/utils/k8s"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+const (
+	// refreshPeriod defines how often the licensing information should be refreshed
```

Comment:
Do we maybe want to make this configurable with a default? I can see leaving that for later though.

Reply:
I'm +1 on keeping that for later. Too many flags :smile:

Reply:
Good. I also thought that we will add it later if we really feel the need.